modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,139 @@
+Place a Blast marker on the detachment.
+
+^*^If you leave the detachment in Portal Reserves, then:
+
+* If your army uses `_(Roll)_` portals, then you must roll on the Reserve Detachment Arrival table again next turn.
+* If your army uses `_(Auto)_` portals, then you must try again next turn.
+
+IMPORTANT: If the detachment does not arrive and enter play before all {portal} units/objectives in your army are destroyed or the game ends, then the detachment counts as destroyed for the purposes of this game.
+footnote:[
+If you're playing a scenario that uses the Army Morale rules, deduct the detachment's Morale Value from your morale total at the end of the Rally phase, as normal for a destroyed detachment.
+]
+footnote:[
+We consider the detachment to be 'destroyed' for the sake of the battle, but not for the war.
+That is, treat the detachment as destroyed for all rules purposes in the current game -- but not for future games, if you're playing a campaign.
+]
+
+To complete the move of units that you can put into play, obey the rules at <<portal-moves>>.
+
+== Example of {portal} in action
+// TODO: This example needs more work, but it's a bit difficult while the structure and logic of the rules above are still in a state of flux.
+
+.Necron Portals
+====
+You have a Necron army which includes, among other things, two different {portal} units:
+
+* A Monolith `WE` unit -- which has `{portal} _(INF, 4+, Auto)_`.
+* A Warbarque `WE` unit -- which has `{portal} _(INF, {walker}, 4+, Auto)_`.
+
+Your army also includes two detachments that you decide to keep outside of the play area, as Portal Reserves:
+
+* Detachment A, which contains some Necron Warriors (`INF`), Lokhust Destroyers (`INF`), and Tomb Blades (`VEH` with Armour `4+`).
+* Detachment B, which contains some Necron Immortals (`INF`), Canoptek Spyders (`VEH` with `{walker}`), and a C'Tan Shard (`VEH` with `{walker}`).
+
+As the portals in the Necron army list are `_(Auto)_` portals, detachments A and B will both arrive in turn 2.
+
+When they arrive, you can use detachments A and B with the {portal} units as follows:
+
+* All of the units in detachment A can pass through either portal.
+So, you can use either {portal} unit to disembark detachment A, provided the portal is available.
+* The {walker} units in detachment B can pass through only the Warbarque's portal.
+So, you must either:
+** Use the Warbarque {portal} unit to disembark this detachment, provided the portal is available.
+** Use the Monolith {portal} unit to disembark this detachment, provided the portal is available -- but in this case, the Canoptek Spyders and the C'Tan Shard are *destroyed* and you must place a Blast marker on the Immortals units that form what's left of the detachment.
+// TODO: In this specific example, assuming both portals are available in turn 2, can the player deliberately choose to take a loss and disembark detachment B through the Monolith's portal and A through the Warbarque's portal?
```

Comment:
I can't see anything that would stop it, and you've covered it in your example.

Reply:
Cool. In that case I think we can simply remove my query.
```suggestion
```